src/Analyzer.Types/TemplateContext.cs

```diff
@@ -27,9 +27,9 @@ public class TemplateContext
         public bool IsBicep { get; set; }
 
         /// <summary>
-        /// Source map that maps ARM JSON back to source Bicep
+        /// TODO Source map that maps ARM JSON back to source Bicep
```

Comment:
is this tracked/blocking?

Reply:
Nope just need to update this as it's now bicep metadata (source map and module info). Good catch.